Source/Extensions/Blazorise.Charts.Zoom/ChartZoom.razor.cs

```diff
@@ -16,23 +17,139 @@ public partial class ChartZoom<TItem> : ChartPlugin<TItem, JSChartZoomModule>
 
     /// <inheritdoc/>
     protected override JSChartZoomModule CreatePluginJsModule()
-        => new( JSRuntime, VersionProvider, BlazoriseOptions );
+    {
+        DotNetObjectRef ??= DotNetObjectReference.Create( new ChartZoomAdapter<TItem>( this ) );
```

Comment:
What's the logic behind the `??=` operator here?  
It suggests a scenario where `CreatePluginJsModule` might be called multiple times. But that's not the case, right?  
If that were true, it would imply `OnParentChartInitialized` is called more than once. That would be an exceptional situation, so throwing an exception here should be the correct approach. (otherwise it would fail silently)

Reply:
Not sure what you mean. `??=` is just a shorthand for null check and then assigning the new value to a var.

Reply:
Wouldn't that break the pre-rendering since initialize is called twice in SSR?